framework/src/modules/chain/migrations/sql/20191004134805_simplify_transaction_asset_schema.sql

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
This file no longer needed?

Reply:
This one is needed. The other one is not.